pkg/tool/kubectl.go

```diff
@@ -184,6 +184,10 @@ func (k Kubectl) GetPods(args ...string) ([]string, error) {
 	return strings.Fields(pods), nil
 }
 
+func (k Kubectl) GetEvents(namespace string) error {
+	return k.exec.RunProcess("kubectl", "get", "events", "--namespace", namespace)
```

Comment:
Adding `-o wide` also includes a `COUNT` column which could be useful.

```suggestion
	return k.exec.RunProcess("kubectl", "get", "events", "--output", "wide", "--namespace", namespace)
```